test/solidity/block/BlockInfo.js

```diff
@@ -59,34 +60,34 @@ describe('@solidityequiv1 BlockInfo Test Suite', function () {
     expect(coinbase).to.equal('0x0000000000000000000000000000000000000062');
   });
 
-  // Turn off until mirror node issue is resolved: https://github.com/hashgraph/hedera-mirror-node/issues/7036
   it('should get the current block prevrandao using block.prevrandao', async function () {
+    let prevrandao;
     try {
-      await blockInfo.getBlockPrevrando();
+      prevrandao = await blockInfo.getBlockPrevrando();
+      console.log('prevrandao', prevrandao);
```

Comment:
nit: does this log help in any way? If not I'd suggest clean it up.

Reply:
No, sorry, cleaned up.